docker/checkout.bash

```diff
@@ -45,9 +45,9 @@ if [[ "$BRANCH" = "develop" ]]; then
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch $BRANCH
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch $BRANCH
 else
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch feature/ihp_develop_merge
+      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch develop
```

Comment:
I've never understood the logic in this section. The if block above guarantees that $BRANCH == "DEVELOP", yet uses the variable name in its commands instead of just hard-coding the word "develop".  But in the else block, where $BRANCH could be anything, we hard-code the word "develop" so that $BRANCH is simply ignored.  Seems totally backwards.  However, this isn't the place to argue the legacy logic, so I'll let it go.  But if you have some insight, I'd love to hear it.